dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go

```diff
@@ -357,6 +357,7 @@ func generateTarball(opts *GenerateTarballOpts) error {
 		run("adding Alluxio FUSE jar", "mv", fmt.Sprintf("integration/fuse/target/alluxio-integration-fuse-%v-jar-with-dependencies.jar", version), filepath.Join(dstPath, "integration", "fuse", fmt.Sprintf("alluxio-fuse-%v.jar", version)))
 		// Generate Helm templates in the dstPath
 		run("adding Helm chart", "cp", "-r", filepath.Join(srcPath, "integration/kubernetes/helm-chart"), filepath.Join(dstPath, "integration/kubernetes/helm-chart"))
+		run("adding tool dir", "cp", "-r", filepath.Join(srcPath, "integration/tool"), filepath.Join(dstPath, "integration/tool"))
```

Comment:
it is not preferred to copy entire directories. can you add the two new files as part of the array of strings for the variable `pathsToCopy` around line 200 in the same file?